tests/test_sklearn_gaussian_process.py

```diff
@@ -99,11 +103,18 @@ def remove_dim1(self, arr):
 
     def check_outputs(self, model, model_onnx, Xtest,
                       predict_attributes, decimal=5,
-                      skip_if_float32=False):
+                      skip_if_float32=False, disable_optimisation=True):
         if predict_attributes is None:
             predict_attributes = {}
         exp = model.predict(Xtest, **predict_attributes)
-        sess = InferenceSession(model_onnx.SerializeToString())
+        if disable_optimisation and GraphOptimizationLevel is not None:
+            opts = SessionOptions()
+            opts.graph_optimization_level = (
+                GraphOptimizationLevel.ORT_ENABLE_ALL)
```

Comment:
I am a little confused here? 

if disable_opt:
   GraphOptimizationLevel.ORT_ENABLE_ALL 
or what you want is :
GraphOptimizationLevel.ORT_DISABLE_ALL

Reply:
Right, I tried ENABLE to check that the build was not failing. I forgot to put it back. I'll need to merge PR #516 to fully test it. But that PR fails for the same reason as the nightly build.